easybuild/easyblocks/o/openblas.py

```diff
@@ -57,6 +120,17 @@ def build_step(self):
                 build_parts += ['re_lapack']
         build_parts += ['shared']
 
+        # If we're doing either a dynamic build or utilizing optarch,
+        # strip march from all environment variables except the EBVAR-prefixed ones.
+        # For dynamic builds we should ignore optarch completely and for optarch-set builds
+        # we need to adhere to TARGET and not march.
+        if self._dynamic_target is True or self._optarch_architecture is True:
```

Comment:
In which case can `self._optarch_architecture` be True? It is either False or a string isn't it?

Reply:
Hm, yeah. I think this was a suggested change from the editor of best practices for python. There was something odd about that. I'll check it out.